photos/db.go

```diff
@@ -258,8 +258,11 @@ func listPhotos(tx *sql.Tx, userID int, photoIDs *[][]byte) error {
 	if err != nil {
 		return err
 	}
-	const selectSQL = `
-SELECT id, caption, commentCount, latitude, longitude, timestamp FROM photos WHERE userID = $1 ORDER BY timestamp DESC LIMIT 100`
+	const selectSQL = `SELECT commentID, userID, message, timestamp FROM comments ` +
```

Comment:
I'm happy if someone wants to take it over - I've got some loose ends to tie up. @RaduBerinde, would you mind?